### PR TITLE
Fix itemChoice right content display

### PR DESCRIPTION
**Pull request by nickoola**

### CODE REVIEW

**Comment by pith** [comment]
Can you add a screenshot please 🙂

File: src/itemChoice/index.tsx [hunk 1]
```diff
@@ -123,10 +123,12 @@ class ItemChoice extends PureComponent <ItemChoiceProps> {
           </div>
         )}
         {children}
-        {rightAddon && (
-          <div className={cc(prefix({ 'itemChoice-rightAddon': true }))}>{rightAddon}</div>
-        )}
-        {rightIcon}
+        <div className="kirk-itemChoice-right">
+          {rightAddon && (
```
Comment (wakooka) [inline src/itemChoice/index.tsx]: Is it intended to have an empty `div` in some cases ?

**Comment by nickoola** [comment]
capture d ecran 2018-06-19 a 11 54 39
capture d ecran 2018-06-19 a 11 54 13
capture d ecran 2018-06-19 a 11 53 55
capture d ecran 2018-06-19 a 11 53 39

File: src/itemChoice/index.tsx (hunk 1 above)
Comment (nickoola) [reply src/itemChoice/index.tsx]: It's never empty, we always have the arrow as default.

Comment (nickoola) [reply src/itemChoice/index.tsx]: rightAddon is when we want to add a content near the arrow.

Comment (wakooka) [reply src/itemChoice/index.tsx]: right, my bad, I didn't see `{rightIcon}` underneath

**Pull request merged**